diff --git a/scripts/build/configure/termux_step_configure_autotools.sh b/scripts/build/configure/termux_step_configure_autotools.sh
index 5e209c2e2b..86f582af45 100644
--- a/scripts/build/configure/termux_step_configure_autotools.sh
+++ b/scripts/build/configure/termux_step_configure_autotools.sh
@@ -6,10 +6,10 @@ termux_step_configure_autotools() {
 		ENABLE_STATIC=""
 	fi
 
-	local DISABLE_NLS="--disable-nls"
-	if [ "$TERMUX_PKG_EXTRA_CONFIGURE_ARGS" != "${TERMUX_PKG_EXTRA_CONFIGURE_ARGS/--enable-nls/}" ]; then
-		# 如果软件包明确启用了 nls，则不要禁用它（例如 gettext 本身）
-		DISABLE_NLS=""
+	local DISABLE_NLS=""
+	if [ "$TERMUX_PKG_EXTRA_CONFIGURE_ARGS" != "${TERMUX_PKG_EXTRA_CONFIGURE_ARGS/--disable-nls/}" ]; then
+		# 如果软件包明确禁用了 nls，则禁用它
+		DISABLE_NLS="--disable-nls"
 	fi
 
 	local ENABLE_SHARED="--enable-shared"
diff --git a/scripts/build/termux_download.sh b/scripts/build/termux_download.sh
old mode 100755
new mode 100644
index 953a55a1bc..ff01fbef87
--- a/scripts/build/termux_download.sh
+++ b/scripts/build/termux_download.sh
@@ -9,6 +9,13 @@ termux_download() {
 	local DESTINATION="$2"
 	local CHECKSUM="${3:-SKIP_CHECKSUM}"
 
+	# 自动替换下载URL为镜像加速
+	# GitHub镜像加速
+	if [[ "$URL" =~ ^https?://github\.com/ ]]; then
+		URL="https://gh.llkk.cc/${URL}"
+		echo "使用GitHub镜像加速下载"
+	fi
+
 	if [[ "$URL" =~ ^file://(/[^/]+)+$ ]]; then
 		local source="${URL:7}" # 移除 `file://` 前缀
 
diff --git a/scripts/build/termux_step_massage.sh b/scripts/build/termux_step_massage.sh
index c60eb1e220..8dede6aec5 100644
--- a/scripts/build/termux_step_massage.sh
+++ b/scripts/build/termux_step_massage.sh
@@ -29,7 +29,7 @@ termux_step_massage() {
 	rm -f share/icons/hicolor/icon-theme.cache
 
 	# Remove locale files we're not interested in:
-	rm -Rf share/locale
+	# rm -Rf share/locale
 
 	# Remove ldconfig cache:
 	rm -f glibc/etc/ld{,32}.so.cache
diff --git a/scripts/build/termux_step_setup_variables.sh b/scripts/build/termux_step_setup_variables.sh
index 9452699777..79e7c77610 100644
--- a/scripts/build/termux_step_setup_variables.sh
+++ b/scripts/build/termux_step_setup_variables.sh
@@ -5,7 +5,7 @@ termux_step_setup_variables() {
 	: "${TERMUX_FORCE_BUILD:="false"}"
 	: "${TERMUX_FORCE_BUILD_DEPENDENCIES:="false"}"
 	: "${TERMUX_INSTALL_DEPS:="false"}"
-	: "${TERMUX_PKG_MAKE_PROCESSES:="$(nproc)"}"
+	: "${TERMUX_PKG_MAKE_PROCESSES:="8"}"
 	: "${TERMUX_PKGS__BUILD__RM_ALL_PKGS_BUILT_MARKER_AND_INSTALL_FILES:="true"}"
 	: "${TERMUX_PKGS__BUILD__RM_ALL_PKG_BUILD_DEPENDENT_DIRS:="false"}"
 	: "${TERMUX_PKG_API_LEVEL:="24"}"
